modules/cert-manager-fips-support.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cert-manager-fips-support_{context}"]
+= About FIPS compliance for {cert-manager-operator}
+
+For many {product-title} customers, regulatory readiness, or compliance, on some level is required before any systems can be put into production. That regulatory readiness can be imposed by national standards, industry standards or the organization's corporate governance framework. The {cert-manager-operator} now supports Federal Information Processing Standards (FIPS) compliance, a crucial requirement for highly secure environments to ensure the use of approved cryptographic technologies within the cluster.
```

Comment:
Looks good to me, except maybe you can add "FIPS 140" added specifically.
Also, you can add that with the new changes cert-manager now uses NIST validated crypto libraries.

Reply:
Additionally, I found a slide deck with some FIPS wordings
maybe you could refer that as well:  [Red Hat Internal] https://docs.google.com/presentation/d/1Rt0AoUK12__2CQgWM_Qk9A7GPEqKMO4mQZhW3mA4pU8/edit#slide=id.g2696a798143_1_0
since we're (cert-manager-operator) technically an OpenShift Layered product.

Reply:
Great. Let me rephrase it.